vpp-manager/startup/startup.go

```diff
@@ -115,13 +115,21 @@ func parseEnvVariables(params *config.VppManagerParams) (err error) {
 		hooks.RegisterBashHook(hooks.BEFORE_IF_READ, conf)
 	}
 
+	// NetworkManager wipes out DNS config when the uplink interface
+	// is moved. We counter it by telling it to leave the DNS config alone.
+	hooks.RegisterBashHook(hooks.BEFORE_VPP_RUN, hooks.FIX_DNS_SCRIPT)
+
 	// Add default hook if none specified
 	for _, hookName := range []string{hooks.VPP_RUNNING, hooks.VPP_DONE_OK, hooks.VPP_ERRORED} {
 		if hooks.HookCount(hookName) == 0 {
 			hooks.RegisterBashHook(hookName, hooks.DEFAULT_RESTART_SCRIPT)
```

Comment:
Having a single script would also allow user override with this function (we should probably add `BEFORE_VPP_RUN` to the lot of hooks having our script as the default)